src/content/docs/setup/configuration/aem-prerender.mdx

```diff
@@ -0,0 +1,552 @@
+        A --> D
+        B --> D
+        C --> D
+    end
+
+    E["public/ folder<br/>PDPs served publicly"]:::public
+    F["Edge Delivery Services<br/>fetches HTML from public blob store folder"]:::helix
+    
+    subgraph subGraph1["<strong>Product Page</strong>"]
+        direction TB
+        G["Embedded markup with product data<br/>and attributes, injected ahead-of-time"]:::embedded
+        H["Description"]:::prerendered
+        I["Product image carousel"]:::prerendered
+        J["Price<br/>Stock qty"]:::dynamic
+        K["More<br/>Customer-defined fields"]:::dynamic
+    end
+    
+    subgraph subGraph2["<strong>Consumers</strong>"]
+        direction TB
+        L["SEO / GEO"]
+        M["AI Scrapers<br/>(read markup, no js)"]
+        N["Googlebot<br/>(reads JSON-LD, Rendered page)"]
+        O["Other crawlers<br/>(read markup, no js)"]
+    end
+
+    P["Catalog service<br/>(high frequency data)"]:::api
+    Q["API for price retrieval"]:::api
+
+    D -->|Order preview/publish/unpublish| E
+    E --> F
+    F -->|Edge Delivery serves the final page| subGraph1
+    subGraph1 --> subGraph2
+    G --> H
+    G --> I
+    H --> subGraph2
+    I --> subGraph2
+    J --> P
+    J --> Q
+    K --> P
+    K --> Q
+
+    classDef appBuilder fill:#e3f2fd,stroke:#2962FF,color:#2962FF
+    classDef storage fill:#fff3e0,stroke:#f57c00
+    classDef public fill:#f3e5f5,stroke:#7b1fa2
+    classDef helix fill:#e8f5e9,stroke:#388e3c
+    classDef embedded fill:#fff9c4,stroke:#f57f17
+    classDef prerendered fill:#c8e6c9,stroke:#388e3c
+    classDef dynamic fill:#ffcdd2,stroke:#c62828
+    classDef api fill:#fce4ec,stroke:#c2185b
+`}
+/>
+
+<Aside type="note" title="Catalog Size Considerations">
+For catalogs exceeding 50,000 products, consult the <Link href="https://www.aem.live/docs/limits" text="AEM indexing limits" /> documentation for sitemap splitting strategies and scaling considerations.
+</Aside>
+
+## Installation and configuration
+
+The <Link href="https://github.com/adobe-rnd/aem-commerce-prerender" text="AEM Commerce Prerender repository" /> provides complete installation instructions. The following overview highlights the key configuration steps and values you'll need.
+
+<Steps>
+
+1. **Clone the repository** and install dependencies:
+   ```bash
+   git clone https://github.com/adobe-rnd/aem-commerce-prerender.git
+   cd aem-commerce-prerender
+   npm install
+   ```
+
+2. **Run the setup wizard** to generate your `.env` configuration:
+   ```bash
+   npm run setup
+   ```
+   The wizard prompts for your App Builder workspace selection and Adobe I/O authentication.
+
+3. **Configure environment variables** in the generated `.env` file with your storefront-specific values:
+
+   <TableWrapper nowrap={[0]}>
+
+   | Variable | Description | Example |
+   |----------|-------------|---------|
+   | `CATALOG_SERVICE_URL` | Your Adobe Commerce Catalog Service endpoint | `https://catalog-service.adobe.io/graphql` |
```

Comment:
For Adobe Commerce Optimizer, this URL is different. See  endpoints:  https://experienceleague.adobe.com/developer/commerce/storefront/setup/configuration/commerce-configuration/#endpoints